lib/createExplorer.js

```diff
@@ -51,7 +54,7 @@ module.exports = function (options) {
       });
   }
 
-  function searchDirectory(directory) {
+  function searchDirectory(directory, lastSearchDirectory) {
```

Comment:
I'm wondering about other options. The problem is: if you start search directory one last time, only for the home, the `nextDirectory` variable WILL have a value and will be DIFFERENT than `stopDir`, so, it will keep searching on other folders. The goal is to search on user's home only once and don't go to the next directory. We can change it to ask if should go down to the next directory or not too.